CentralComputing/Pod.cpp

```diff
@@ -39,6 +45,8 @@ int main() {
 
```

Comment:
At some point I will include setup for device tree stuff here.  Also probably ought to add a simple compile time flag that enables this extra setup to only run on a BBB